src/plugins/BunqLayoutPlugin/components/SidebarItem/SidebarItem.js

```diff
@@ -67,7 +67,7 @@ class SidebarItem extends React.Component {
 
     /**
      */
-    componentDidMount () {
+    componentDidUpdate () {
         if (!this.state.isApiEndpoint) {
```

Comment:
in case of `componentDidUpdate` you want to make a comparison to the previous state. this method runs on every update on anything on state / props, and whenever `!this.state.isApiEndpoint` holds that code block will be called causing unnecessary rerenders - not sure whether this one would cause an infinite rerender or sth but I would check the behaviour with some logs.

the function signature is `componentDidUpdate(prevProps, prevState)`, this should work nicely if you do `if (prevState.isApiEndpoint && !this.state.isApiEndpoint) { `

Reply:
if the result of `this.props.bunqSelectors.initializeScrollToTopic(this.state.tag);` doesn't affect this component i.e. props sent from the parent will remain the same it should be fine, otherwise it will infinitely trigger a rerender by itself. there might be some props sent down that might not be obvious but still trigger a rerender, but if you already checked or if you are sure it's fine :)